documentation/modules/rn-2.3.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-23_{context}"]
+= {project-full} 2.3
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Setting the VddkInitImage path is part of the procedure of adding VMware provider.
+
+In the web console, you enter the VddkInitImage path when adding a VMware provider. Alternatively, from the CLI, you add the VddkInitImage path to the `Provider` CR for VMware migrations.
+
+.The `StorageProfile` resource needs to be updated for a non-provisioner storage class
+
+You must update the `StorageProfile` resource with `accessModes` and `volumeMode` for non-provisioner storage classes such as NFS. The documentation includes a link to the relevant procedure.
+
+[id="new-features-and-enhancements-22_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.{project-short} {project-version} supports warm migration from {rhv-short}
+
+You can use warm migration to migrate VMs from both VMware and {rhv-short}.
+
+.The minimal sufficient set of privileges for VMware users is established
+
+VMware users do not have to have full `cluster-admin` privileges to perform a VM migration. The minimal sufficient set of user's privileges is established and documented.
+
+.{project-short} documentation is updated with instructions on using hooks
+
```

Comment:
period needed.

Reply:
I think so too, but for some reason Avital told me that there should not be periods in these headings, although they are complete sentences.

Reply:
Is line 38 a heading? If so, then no period is needed. No punctuation in headings; Avital's 100% correct.

Reply:
@RichardHoch  These are sort of headings but not exactly headings. They are more like summaries of the issues. That's why they are complete sentences. I would put periods there, but Avital disagreed.

Reply:
They are headings. it's nice you made them sentences (seriously) but no punctuation there.